code/modules/mob/living/carbon/xenomorph/castes/behemoth/castedatum_behemoth.dm

```diff
@@ -10,35 +10,31 @@
 	wound_type = "behemoth"
 
 	// *** Melee Attacks *** //
-	melee_damage = 23
+	melee_damage = 30
```

Comment:
I'm not really looking at the balance right now, but why is bemo doing better damage than a rav, while being more tanky, and being stager immune and still having all the funny abilities?